specification/containerservice/Fleet.Management/fleetnamespace.tsp

```diff
@@ -92,84 +100,97 @@ model ResourceQuota {
+  Never: "Never",
 
-  @doc("Deny all egress traffic from the managed namespace")
-  DenyAll: "DenyAll",
+  @doc("If there is an identical managed namespace, take over the existing managed namespace to be managed by ARM.")
+  IfIdentical: "IfIdentical",
+
+  @doc("Always take over the existing managed namespace to be managed by ARM, even if it is not identical.")
+  Always: "Always",
 
   string,
 }
 
-@doc("The profile of the fleet managed namespace.")
-@added(Versions.v2025_03_02_naar)
-model FleetManagedNamespaceProfile {
-  @doc("Action if Kubernetes managed namespace with the same name already exists.")
-  adoptionPolicy?: AdoptionPolicy;
-
-  @doc("Delete options of a fleet managed namespace.")
-  deletePolicy?: DeletePolicy;
+@doc("Delete options of a managed namespace.")
+union DeletePolicy {
+  @doc("Delete ARM resource but keep k8s managed namespace.")
+  Keep: "Keep",
 
-  #suppress "@azure-tools/typespec-azure-resource-manager/arm-no-record" "Records is the most semantically correct choice for this use-case"
-  @doc("The labels for the fleet managed namespace.")
-  labels?: Record<string>;
+  @doc("Delete both ARM resource and k8s managed namespace.")
+  Delete: "Delete",
 
-  #suppress "@azure-tools/typespec-azure-resource-manager/arm-no-record" "Records is the most semantically correct choice for this use-case"
-  @doc("The annotations for the fleet managed namespace.")
-  annotations?: Record<string>;
+  string,
+}
 
-  @doc("The default resource quota for the fleet managed namespace.")
-  defaultResourceQuota?: ResourceQuota;
+@added(Versions.v2025_03_02_naar)
+@doc("The propagation to be used for provisioning the namespace among the fleet.")
+model PropagationPolicy {
+  @doc("The type of the policy to be used. Default is Placement.")
+  type?: PropagationType;
 
-  @doc("The default network policy for the fleet managed namespace.")
-  defaultNetworkPolicy?: NetworkPolicy;
+  @doc("The propagation profile to be used for the propagation. Default is PlacementProfile.")
+  propagationProfile?: PropagationProfile;
 }
 
 @added(Versions.v2025_03_02_naar)
-@doc("Status information for the fleet managed namespace.")
-model FleetManagedNamespaceStatus {
-  @visibility("read")
-  @doc("The last operation ID for the fleet managed namespace")
-  lastOperationId?: string;
+@doc("How the namespace will be provisioned among the fleet members.")
+union PropagationType {
+  @doc("Using ClusterResourcePlacement.")
+  Placement: "Placement",
 
-  @visibility("read")
-  @doc("The last operation error of the fleet managed namespace")
-  lastOperationError?: Azure.ResourceManager.Foundations.ErrorDetail;
+  string,
 }
 
-@doc("A fleet managed namespace.")
 @added(Versions.v2025_03_02_naar)
-model FleetManagedNamespaceProperties {
-  @visibility("read")
-  @doc("The status of the last operation.")
-  provisioningState?: FleetManagedNamespaceProvisioningState;
+@doc("The available Propagation profiles.")
+union PropagationProfile {
```

Comment:
I don't straightforwardly understand what this is doing - please could you show me a json example?

I think what I'm expecting to see is as follows -- is this what it's doing?

```json
"propagationPolicy": {
  "type": "Placement",
  "placementProfile": {
    "defaultClusterResourcePlacement": {}
  }
}
```